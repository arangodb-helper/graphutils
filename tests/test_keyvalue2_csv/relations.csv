_key,_from,_to
"1",profiles/444,profiles/222
"2",profiles/111,profiles/444
"3",profiles/999,profiles/444
"4",profiles/555,profiles/333
"5",profiles/111,profiles/222
"6",profiles/444,profiles/999
"7",profiles/777,profiles/999
"8",profiles/222,profiles/444
"9",profiles/555,profiles/555
"10",profiles/888,profiles/777
