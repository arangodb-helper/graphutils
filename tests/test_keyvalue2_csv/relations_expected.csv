_key,_from,_to
44:1:22,profiles/44:444,profiles/22:222
11:2:44,profiles/11:111,profiles/44:444
99:3:44,profiles/99:999,profiles/44:444
55:4:33,profiles/55:555,profiles/33:333
11:5:22,profiles/11:111,profiles/22:222
44:6:99,profiles/44:444,profiles/99:999
77:7:99,profiles/77:777,profiles/99:999
22:8:44,profiles/22:222,profiles/44:444
55:9:55,profiles/55:555,profiles/55:555
88:10:77,profiles/88:888,profiles/77:777
